### PR TITLE
Update the layouts to the Android and iOS dashboards

### PR DESCRIPTION
**Pull request by ksiegler1**
Made some changes to the layouts. I had to comment out the four lines in a Firefox accounts lookml file order to validate these changes.

### CODE REVIEW

File: firefox_accounts/firefox_accounts.model.lkml [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 include: "//looker-hub/firefox_accounts/explores/*"
-include: "//looker-hub/firefox_accounts/dashboards/*"
-include: "views/*"
-include: "explores/*"
-include: "dashboards/*"
\ No newline at end of file
+# include: "//looker-hub/firefox_accounts/dashboards/*"
+# include: "views/*"
+# include: "explores/*"
+# include: "dashboards/*"
```
Comment (ksiegler1) [inline firefox_accounts/firefox_accounts.model.lkml:7]: @fbertsch these are the lines I commented out - tagging you for context around the errors I encountered earlier.

Comment (fbertsch) [reply firefox_accounts/firefox_accounts.model.lkml:7]: Great, sounds good. I made similar changes in https://github.com/mozilla/looker-spoke-default/pull/102 which you may have to rebase on top of.

File: duet/dashboards/mobile_android_country.dashboard.lookml [hunk 2]
```diff
@@ -468,10 +465,264 @@
...
+  - name: 'Step 3: First Seen in Internal Telemetry'
+    type: text
+    title_text: 'Step 3: First Seen in Internal Telemetry'
+    subtitle_text: ''
+    body_text: The count of new client IDs measured in our internal telemetry. This
+      is our internal indication of how many acquisitions we have gotten during the
+      time period of interest. The jump from Step 2 to Step 3 is the installation
+      from Google Play Store to opening the app for the first time - these events
+      may happen on the same day, or a user could have installed from the Play Store
+      during a previous time period and opening the app for the first time during
```
Comment (acmiyaguchi) [inline duet/dashboards/mobile_android_country.dashboard.lookml:541]: nit: tense

```suggestion
      during a previous time period and opened the app for the first time during
```

**Review by acmiyaguchi — APPROVED** [review]
These changes look fine to me. I think dropping the firefox account changes should allow the branch to merge in cleanly.

**Comment by acmiyaguchi** [comment]
Thanks for resolving the conflicts Kimmy!

**Pull request merged**